api/main_endpoints/routes/Printer.js

```diff
@@ -74,29 +78,42 @@ router.get('/healthCheck', async (req, res) => {
 });
 
 router.post('/sendPrintRequest', upload.single('chunk'), async (req, res) => {
+  let totalFileSize = 0;
+  const { copies, sides, id } = req.body;
+  const action = AuditLogActions.PRINT_PAGE;
+
   if (!checkIfTokenSent(req)) {
     logger.warn('/sendPrintRequest was requested without a token');
     return res.sendStatus(UNAUTHORIZED);
   }
-  if (!await decodeToken(req)) {
+  const user = decodeToken(req);
+  if (!user) {
     logger.warn('/sendPrintRequest was requested with an invalid token');
     return res.sendStatus(UNAUTHORIZED);
   }
-  if (!PRINTING.ENABLED) {
-    logger.warn('Printing is disabled, returning 200 and dummy print id to mock the printing server');
-    return res.status(OK).send({ printId: null });
-  }
 
-  const dir = path.join(__dirname, 'printing');
-  const { totalChunks, chunkIdx } = req.body;
+  const details = {
+    copies: parseInt(copies),
+    sides: sides,
```

Comment:
```suggestion
    sides,
```
when key = value we only need to write it once

Reply:
Done!